build.mjs

```diff
@@ -1,5 +1,5 @@
-import fs from 'fs';
-import path from 'path';
+import fs from 'node:fs';
+import path from 'node:path';
```

Comment:
Just curious - how come these need the `node:` prefix?

Reply:
Airbags and safety belts. Good habit to avoid name clashes.